azure-pipelines.yml

```diff
@@ -40,8 +40,13 @@ stages:
         displayName: "Install NuGet"
 
       - script: nuget restore -LockedMode -ConfigFile "analyzers\NuGet.Config" $(solution)
+        displayName: "NuGet Restore"
 
-      - powershell: .\scripts\build\set-azp-project-version-variable.ps1
+      - powershell: .\scripts\build\store-azp-variables.ps1
+        displayName: "Store AZP Variables"
+
+      - powershell: .\scripts\build\set-azp-variables.ps1
+        displayName: "Set AZP Variables"
```

Comment:
Set `displayName` to avoid `CmdLine`, `PowerShell`, `PowerShell` default job names.

Reply:
thanks! that was annoying indeed